### ISSUE TITLE
Update flutter 3.24 Packaging apk error

### ISSUE THREAD

**Issue by 1xiaocainiao** [leanflutter/flutter_svprogresshud#18]
The mistake is as follows

 What went wrong:
Execution failed for task ':flutter_svprogresshud:verifyReleaseResources'.
> A failure occurred while executing com.android.build.gradle.tasks.VerifyLibraryResourcesTask$Action
   > Android resource linking failed
     ERROR:/Users/qqq/Desktop/app/build/flutter_svprogresshud/intermediates/merged_res/release/values/values.xml:194: AAPT: error: resource android:attr/lStar not found.

How to solve it，please

**Comment by Syaba999** [comment]
Use [this](https://github.com/Syaba999/flutter_svprogresshud) fork

in `pubspec.yaml`
```
  flutter_svprogresshud:
    git:
      url: https://github.com/Syaba999/flutter_svprogresshud
```

**Comment by 1xiaocainiao** [comment]
> https://github.com/Syaba999/flutter_svprogresshud

ok, thanks

**Comment by pastelcode** [comment]
> Use [this](https://github.com/Syaba999/flutter_svprogresshud) fork
> 
> in `pubspec.yaml`
> 
> ```
>   flutter_svprogresshud:
>     git:
>       url: https://github.com/Syaba999/flutter_svprogresshud
> ```

Thank you!! Open a PR for this fix please 🙏🏻.

**Comment by lijy91** [comment]
Fixed, please update to 1.1.0
